src/server/services/training.service.ts

```diff
@@ -1,11 +1,18 @@
 import { TrainingStatus } from '@prisma/client';
+import { calcBuzzFromEta, calcEta } from '~/components/Resource/Forms/Training/TrainingCommon';
```

Comment:
Please consider moving these helper methods into the `~/utils` directory, this way we don't mix client side declarations with server side

Reply:
cool ill move it there